.changeset/blue-carpets-jam.md

```diff
@@ -0,0 +1,21 @@
+---
+"@changesets/ghcommit": major
+---
+
+Refactor & clean up options for multiple functions
+
+- For `commitFilesFromDirectory`:
+  - Rename `workingDirectory` to `cwd` for consistency across repos,
+    and utils like `exec`
+  - Make `cwd` a required argument
```

Comment:
It would be OK to make this optional and just default to `process.cwd()`. In `changesets/action` we'd still provide it explicitly though.

Reply:
Yeah, decided being explicit was probably better anyway, given how easy it is to use `process.cwd()` if that's what people wanted. 😄